src/components/modules/keyboard.js

```diff
@@ -71,9 +71,27 @@ export default class Keyboard extends Module {
      */
     enterPressed(event) {
 
+        let currentBlock = this.Editor.BlockManager.currentBlock,
+            toolsConfig = this.config.toolsConfig[currentBlock.name];
+
         /**
-         * @todo check Tool's configuration for allowLinebreaks property
+         * Check if configuration for enableLineBreaks property true
```

Comment:
Don't handle Enter keydowns when Tool sets enableLineBreaks to true. Uses for Tools like `<code>` where line breaks should be handled by default behaviour.